packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,35 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### AutoScaling
+
+You can use Application AutoScaling to automatically configure the provisioned concurrency for your functions. AutoScaling can be set to track utilization or be based on a schedule. To configure AutoScaling on a function alias:
+
+```ts
+const alias = new lambda.Alias(stack, 'Alias', {
+  aliasName: 'prod',
+  version,
+});
+
+// Create AutoScaling target
+const as = alias.addAutoScaling({ maxCapacity: 50 })
+
+// Configure Target Tracking
+as.scaleOnUtilization({
+  utilizationTarget: 0.5,
+});
+
+// Configure Scheduled Scaling
+as.scaleOnSchedule('ScaleUpInTheMorning', {
+  schedule: appscaling.Schedule.cron({ hour: '8', minute: '0'}),
+  minCapacity: 20,
+});
+```
+
+[Example of Lambda AutoScaling usage](test/integ.autoscaling.ts)
```

Comment:
This link is not going to work when rendered in the [reference documentation](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-lambda-readme.html).

However, if you switch to using a `.lit.ts` extension, this will work in both places. See [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/README.md#breaking-up-methods-and-resources-across-stacks) for an example.